web-registry/src/pages/ProjectMetadata/ProjectMetadata.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import { ProjectMetadataForm } from '../../components/organisms';
+import {
+  EditFormTemplate,
+  OnboardingFormTemplate,
+} from '../../components/templates';
+import {
+  useProjectByIdQuery,
+  useShaclGraphByUriQuery,
+  useUpdateProjectByIdMutation,
+} from '../../generated/graphql';
+import { ProjectMetadataLD } from '../../generated/json-ld';
+import { getProjectShapeIri } from '../../lib/rdf';
+import { useProjectEditContext } from '../ProjectEdit';
+import useProjectMetadataSave from './hooks/useProjectMetadataSave';
+import useProjectMetadataSubmit from './hooks/useProjectMetadataSubmit';
+
+const ProjectMetadata: React.FC = () => {
+  const { projectId } = useParams();
+  const { isEdit } = useProjectEditContext();
+  const [updateProject] = useUpdateProjectByIdMutation();
+  const { data } = useProjectByIdQuery({
+    variables: { id: projectId },
+    fetchPolicy: 'cache-and-network',
+  });
+  const project = data?.projectById;
+  const creditClassId = project?.creditClassByCreditClassId?.onChainId;
+  const isVCS = creditClassId === 'C01';
+  let metadata: ProjectMetadataLD | undefined;
+
+  const { data: graphData } = useShaclGraphByUriQuery({
+    skip: !project,
+    variables: {
+      uri: getProjectShapeIri(creditClassId),
+    },
+  });
+
+  if (project?.metadata) {
+    metadata = project.metadata;
+  }
+
+  const saveAndExit = useProjectMetadataSave();
+  const submit = useProjectMetadataSubmit({
+    project,
+    projectId,
+    updateProject,
+  });
+
+  return isEdit ? (
+    <EditFormTemplate>
+      {!isVCS && (
+        <ProjectMetadataForm
+          submit={submit}
+          initialValues={metadata}
```

Comment:
we should probably exclude the fields from the metadata that were actually set as part of previous steps, such as schema:name, regen:projectSize, schema:location, etc.
currently it shows the full metadata and includes those fields
cc/ @erikalogie

Reply:
I agree with that

Reply:
thanks for your feedback @erikalogie 
cc/ @flagrede

Reply:
I've handled your two comments about handling vcs case and filtering keys here: https://github.com/regen-network/regen-web/pull/1003/commits/c54610b513ad7c272154d324ec31c1dd5be986fd

Not sure which keys I should add in addition to the 3 you mentioned. Let me know if I should add others.

Reply:
Basic Info form:
- schema:name
- regen:projectSize

Location form:
- schema:location

Roles form:
- regen:projectDeveloper

Description form:
- schema:description

Media form:
- regen:previewPhoto
- regen:galleryPhotos
- schema:creditText
- regen:videoURL

Reply:
Thanks! I've added them here: https://github.com/regen-network/regen-web/pull/1003/commits/777544cabd734f35889342a3e5dffd2e803e6a84